packages/react-components/react-avatar/src/components/AvatarGroupPopover/useAvatarGroupPopover.tsx

```diff
@@ -45,10 +45,13 @@ export const useAvatarGroupPopover_unstable = (props: AvatarGroupPopoverProps):
     triggerButtonChildren = count > 99 ? '99+' : `+${count}`;
   }
 
+  const shouldRender = React.Children.count(children) > 0;
```

Comment:
Can you use `count` from L23 here to avoid calling `React.Children.count()` potentially twice in this function?